application/src/main/java/org/thingsboard/server/service/notification/channels/MicrosoftTeamsNotificationChannel.java

```diff
@@ -43,16 +43,21 @@
 import java.util.Optional;
 
 @Component
-@RequiredArgsConstructor
 public class MicrosoftTeamsNotificationChannel implements NotificationChannel<MicrosoftTeamsNotificationTargetConfig, MicrosoftTeamsDeliveryMethodNotificationTemplate> {
 
     private final SystemSecurityService systemSecurityService;
 
     @Setter
-    private RestTemplate restTemplate = new RestTemplateBuilder()
-            .setConnectTimeout(Duration.of(15, ChronoUnit.SECONDS))
-            .setReadTimeout(Duration.of(15, ChronoUnit.SECONDS))
-            .build();
+    private RestTemplate restTemplate;
+
+    public MicrosoftTeamsNotificationChannel(SystemSecurityService systemSecurityService) {
+        this.systemSecurityService = systemSecurityService;
+        this.restTemplate = new RestTemplateBuilder()
+                .setConnectTimeout(Duration.of(15, ChronoUnit.SECONDS))
+                .setReadTimeout(Duration.of(15, ChronoUnit.SECONDS))
+                .build();
+        RestTemplateUtil.excludeJackson2XmlHttpMessageConverter(this.restTemplate);
```

Comment:
Create a method for RestTemplate creation and use it everywhere:
`RestTemplateUtil.newRestTemplate()` which modifies the converters list under the hood and returns a builder.